README.md

```diff
@@ -0,0 +1,15 @@
+# COSCRAD Data Ingestion
+
+## Setup
+If using VS Code, install Microsoft's Python, Pylance, and Python Debugger extensions. 
+
+Create a virtual environment:
+> > > python -m venv .env
+
+## Development
+Activate the virtual environment:
+> > > source .venv/bin/activate
```

Comment:
Here it is called `.venv`. Make sure this is consistent with above instructions.

Reply:
done